transmogrifier/sources/json/aardvark.py

```diff
@@ -55,10 +55,10 @@ def get_source_link(
                 - not used by default implementation, but could be useful for subclass
                     overrides
         """
-        links = cls.get_links(source_record, source_record_id)
-        url_links = [link for link in links if link.kind == "Website"]
-        if len(url_links) == 1:
-            return url_links[0].url
+        if links := cls.get_links(source_record):
+            url_links = [link for link in links if link.kind == "Website"]
+            if len(url_links) == 1:
+                return url_links[0].url
```

Comment:
This is a good reminder on how fuzzy things get over time... I don't think we'd ever expect to have multiple `kind=Website` coming back from `gismit` or `gisogm` records?  

If we did, then we might want another scenario handler for when `len() > 1`, but if not, I think it's good as-is.

**UPDATE**: keeping comment, but after looking more closely at `get_links()`, I think we're okay, given that `kind=Website` is only set for a single possible key in the `dct_references_s` JSON/dictionary values.